android/gradle.properties

```diff
@@ -21,6 +21,7 @@
 # $keytool -genkey -v -keystore ./status-keystore.jks -keyalg RSA -keysize 2048 -validity 10000 -alias status
 
 # Version requirements used throughout the Gradle scripts
+kotlinVersion=1.3.11
```

Comment:
why is this needed ?

Reply:
Without it, android builds were failing (specifically `make run-android` command for me). I'll uncomment and will check if it's still reproducible.

Reply:
Yeah, it's still reproducible:
```
A problem occurred configuring project ':react-native-webview'.
> Could not resolve all artifacts for configuration ':react-native-webview:classpath'.
   > Could not find org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50.
     Searched in the following locations:
       - file:/nix/store/2vcnkyzsghmb5l139ndd0kcz2jsjdc5r-status-react-patched-npm-gradle-modules/.m2/repository/org/jetbrains/kotlin/kotlin-gradle-plugin/1.3.50/kotlin-gradle-plugin-1.3.50.pom
       - file:/nix/store/2vcnkyzsghmb5l139ndd0kcz2jsjdc5r-status-react-patched-npm-gradle-modules/.m2/repository/org/jetbrains/kotlin/kotlin-gradle-plugin/1.3.50/kotlin-gradle-plugin-1.3.50.jar
     Required by:
         project :react-native-webview
```